### PR TITLE
feat/improve oss page UI and screenshots 343

### LINKED ISSUE #343: Improve OSS projects page UI & Screenshots
Inconsistent format for different projects and outdated screenshots

https://guide.eoscostarica.io/docs/open-source-projects

what about using cards instead

- [ ] hacer lista de proyectos 
- [ ] actualizar layout (diseño)
- [ ] actulizar contenido

### PR DESCRIPTION
**Pull request by hanakoori01**
### What does this PR do?

- Resolves #343 

### Steps to test
1. Run `yarn start`
1. Go to `localhost:3000`
1. Go to Developers Guidelines > Open Source Projects

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/open-source-projects.md [hunk 1]
```diff
@@ -45,41 +53,67 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+ 
+ <p align="center">
+	<a href="https://eoscostarica.io/">
+		<img src={useBaseUrl('img/OSS_screnshots/EOS_Rate.PNG')} width="100%"/>
+	</a>
+</p>
+
+<p align="center">
+	<a href="https://eoscostarica.io/">
+		<img src={useBaseUrl('img/OSS_screnshots/EOS_Rate_2.PNG')} width="100%"/>
```
Comment (leisterfrancisco) [inline docs/open-source-projects.md]: Would be better if you use a screenshot from https://eosrate.io/block-producers/costaricaeos/rate

File: docs/open-source-projects.md [hunk 2]
```diff
@@ -45,41 +53,67 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
-- **GitHub** : https://github.com/lacchain/eosio.lacchain.net
+ - **GitHub** : [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/lacchain/eosio.lacchain.net)
 
-![LACChain EOSIO](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/lacchain-eosio-homepage.png)
+ ##  LACChain EOSIO Page
+ 
+ <p align="center">
+	<a href="https://eoscostarica.io/">
+		<img src={useBaseUrl('img/OSS_screnshots/lacchain-eosio-homepage.PNG')} width="100%"/>
```
Comment (leisterfrancisco) [inline docs/open-source-projects.md]: Would be better if you use a screenshot from https://eosio.lacchain.net/en/

File: docs/open-source-projects.md [hunk 3]
```diff
@@ -45,41 +53,67 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+	</a>
+</p>
 
-![LACChain EOSIO](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/lacchain-eosio-homepage1.png)
+<p align="center">
+	<a href="https://eoscostarica.io/">
+		<img src={useBaseUrl('img/OSS_screnshots/lacchain-eosio-homepage1.PNG')} width="100%"/>
```
Comment (leisterfrancisco) [inline docs/open-source-projects.md]: Would be better if you use a screenshot from https://eosio.lacchain.net/en/

> You need to scroll

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Please verify the suggestions make sense, I'm not sure if there is a different order in the content.

File: docs/open-source-projects.md (hunk 2 above)
Comment (hanakoori01) [reply docs/open-source-projects.md]: I'm a little confused, the links are the same for both comments.

https://github.com/eoscostarica/guide.eoscostarica.io/pull/399#discussion_r782368085
https://github.com/eoscostarica/guide.eoscostarica.io/pull/399#discussion_r782366666

@leisterfrancisco

File: docs/open-source-projects.md [hunk 4]
```diff
@@ -240,6 +240,24 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+|   Proton   | A full-stack project template to easily launch your project on Proton. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/proton-full-stack-boilerplate) |
+|   Webapp   | A bare-bones react web app featuring UAL for EOSIO authenticators (wallets). | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/webapp-boilerplate) |
+|   IPFS   | A repository to help you get started integrating IPFS into your project. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/ipfs-boilerplate) |
+|   Backend   | NodeJS Backend + Hasura + PGDB with EOSJS and KEOSD wallet. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/backend-boilerplate) |
+|   EOS dapp   | EOSIO Application Boilerplates. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/eos-dapp-boilerplate) |
+|   EOS dapp   | EOSIO Application Boilerplates. | [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/eos-dapp-boilerplate) |
+
+</div>
```
Comment (leisterfrancisco) [inline docs/open-source-projects.md:260]: I like the solution to organize the information, It is more understandable now. I only have one comment, what if the repository link is added in the boilerplate name? The GitHub logo seems to be a little repetitive.

Comment (hanakoori01) [reply docs/open-source-projects.md:260]: @leisterfrancisco So do you think it's better to just keep it in the name only and remove the logo column or keep it all and add the link to the names?

Comment (leisterfrancisco) [reply docs/open-source-projects.md:260]: Remove logo column

**Review by leisterfrancisco — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**